crates/sui-config/src/node.rs

```diff
@@ -206,14 +206,19 @@ pub struct NodeConfig {
     pub enable_db_write_stall: Option<bool>,
 }
 
-#[derive(Clone, Debug, Deserialize, Serialize, Default)]
+#[derive(Clone, Debug, Deserialize, Serialize)]
 #[serde(rename_all = "kebab-case")]
 pub enum ExecutionCacheConfig {
-    #[default]
     PassthroughCache,
-    WritebackCache {
-        max_cache_size: Option<usize>,
-    },
+    WritebackCache { max_cache_size: Option<usize> },
```

Comment:
Could you also document the behavior when this is None? Not sure if it means unbounded cache or with some constant size